legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-logical-plan/src/main/java/org/finos/legend/engine/persistence/components/planner/BitemporalDeltaPlanner.java

```diff
@@ -370,6 +394,23 @@ private Update getMilestoningLogic()
         return UpdateAbstract.of(mainDataset(), updatePairs, milestoningCondition);
     }
 
+    /*
+    ------------------
+    Stage to Stage without Duplicates Logic:
+    ------------------
+    INSERT INTO stage_without_duplicates (ALL_FIELDS_FROM_STAGE)
+        SELECT ALL_FIELDS_FROM_STAGE FROM stage WHERE NOT EXISTS
+            (SELECT * FROM main WHERE stage.digest = main.digest AND openRecordCondition)
+    */
+    private Insert getStageToStageWithoutDuplicates()
```

Comment:
Can we rename method to populateStageWithoutDuplicates() ?

Reply:
As discussed, this is the same naming convention for all the SQLs used in the VLF mode. It specifies source and target. In this case, it means copying data from stage to stageWithoutDuplicates.